modular_skyrat/modules/blueshield/code/modules/jobs/job_types/blueshield.dm

```diff
@@ -19,7 +19,15 @@
 	display_order = JOB_DISPLAY_ORDER_BLUESHIELD
 	outfit = /datum/outfit/job/blueshield
 
-	family_heirlooms = list(/obj/item/book/manual/wiki/security_space_law, /obj/item/clothing/head/beret/sec)
+	family_heirlooms = list(/obj/item/bedsheet/captain, /obj/item/clothing/head/beret/blueshield)
+
+	mail_goodies = list(
+		/obj/item/storage/fancy/cigarette = 10,
+		/obj/item/stack/spacecash/c500, = 3,
+		/obj/item/disk/nuclear/fake = 2,
```

Comment:
I'm not sure we should be giving anyone the ability to get free illegal tech just because they exist and got mail.

Reply:
never knew the disk qualified for that, I'll swap it with the toy

Reply:
and, the chance is super low anyway